ruma-events/tests/redacted.rs

```diff
@@ -263,7 +264,7 @@ fn redacted_custom_event_serialize() {
             && event_type == "m.made.up"
     );
 
-    let x = from_json_value::<Raw<crate::AnyRedactedSyncStateEvent>>(redacted)
+    let x = from_json_value::<EventJson<AnyRedactedSyncStateEvent>>(redacted)
```

Comment:
`EventJson` was renamed `Raw` and moved in #137, you shouldn't be using the deprecated alias for it :)